modyn/config/schema/modyn_config_schema.yaml

```diff
@@ -49,7 +50,9 @@ properties:
       sample_table_unlogged:
         type: boolean
         description: |
-          This configures whether the table storing all samples is UNLOGGED (= high performance) or crash resilient. Defaults to True. For datasets with many samples (such as Criteo), this is recommended for highest insertion performance. In other scenarios, this might not be necessary.
+          This configures whether the table storing all samples is UNLOGGED (= high performance) or crash resilient. 
+          Defaults to True. For datasets with many samples (such as Criteo), this is recommended for highest insertion performance. 
+          In other scenarios, this might not be necessary.
```

Comment:
is this intended?

Reply:
In my opinion, having these kilometre-long descriptions on a single line makes the yaml difficult to read. These two lines were very long and I broke them up with an enter while editing the schema for something else. However, if you prefer the descriptions as they were before I can rollback

Reply:
hmm I will take a look later, it looked weird on Github in the review system, but might actually look ok in the browser

Reply:
yeah this is the reviewing system. it looked (on my macbook) like several random newlines all over the place :D